internal/value/value.go

```diff
@@ -1244,6 +1598,19 @@ type tzDatetimeValue struct {
 	value string
 }
 
+func (v *tzDatetimeValue) CastTo(dst interface{}) error {
+	switch vv := dst.(type) {
+	case *string:
+		*vv = v.value
+		return nil
+	case *[]byte:
+		*vv = []byte(v.value)
```

Comment:
I suggest to force copy bytes for customer, for prevent change internal saved value if internal representation will change to []byte in the feature.